src/main/java/uk/nhs/hee/tis/trainee/credentials/api/IssueResource.java

```diff
@@ -35,14 +41,40 @@
 @RequestMapping("/api/issue")
 public class IssueResource {
 
+  private final GatewayService service;
+
+  IssueResource(GatewayService service) {
+    this.service = service;
+  }
+
+  @PostMapping("/programme-membership")
+  ResponseEntity<String> issueProgrammeMembershipCredential(@RequestBody ProgrammeMembershipDto dto,
+      @RequestParam(required = false) String state) {
+    Optional<URI> credentialUri = service.getCredentialUri(dto, state, "issue.ProgrammeMembership");
```

Comment:
I wonder if it would be worth keeping all our valid scopes in an enum or something?

Reply:
Probably, another place that it would make sense for the interface to define something, i.e. a `getScope` method.